#!/usr/bin/env python
PACKAGE = "wrench_filter"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("hpf_cutoff_hz", double_t, 0, "hpf_cutoff_hz", 1.0, 0.01 , 1000.0)
gen.add("lpf_cutoff_hz", double_t, 0, "lpf_cutoff_hz", 1.0, 0.01 , 1000.0)
gen.add("gain", double_t, 0, "gain", 1.0, 0.0 , 1e3)
gen.add("hpf_gain", double_t, 0, "hpf_gain", 1.0, 0.0 , 1e3)
gen.add("lpf_gain", double_t, 0, "lpf_gain", 1.0, 0.0 , 1e3)
gen.add("rate", double_t, 0, "rate", 100.0, 0.01 , 1e3)
gen.add("max_force", double_t, 0, "max_force", 1e2, 0.0 , 1e4)
gen.add("max_torque", double_t, 0, "max_torque", 1e2, 0.0 , 1e4)

exit(gen.generate(PACKAGE, "wrench_filter", "WrenchFilter"))
